libnl-route-3.sym

```diff
@@ -1250,3 +1250,17 @@ global:
 	rtnl_route_nh_get_encap_mpls_dst;
 	rtnl_route_nh_get_encap_mpls_ttl;
 } libnl_3_5;
+
+libnl_3_7 {
+global:
+	rtnl_nat_set_old_addr;
+	rtnl_nat_get_old_addr;
+	rtnl_nat_set_new_addr;
+	rtnl_nat_get_new_addr;
+	rtnl_nat_set_mask;
+	rtnl_nat_get_mask;
+	rtnl_nat_set_flags;
+	rtnl_nat_get_flags;
+	rtnl_nat_set_action;
+	rtnl_nat_get_action;
```

Comment:
please sort symbols by name. `vim` can do that nicely ( https://vim.fandom.com/wiki/Sort_lines )